Assets/Scripts/PlayerShooting.cs

```diff
@@ -4,25 +4,37 @@
 
 public class PlayerShooting : MonoBehaviour
 {
+    
     public GameObject arrowPrefab;
-    public float speed = 100f;
+    public float shootingStrength = 0f;
+    public float maxShootingStrength = 3000f;
+    
 
     // Start is called before the first frame update
     void Start()
     {
-     
+        
     }
 
     // Update is called once per frame
     void Update()
     {
-        if (Input.GetMouseButtonDown(0))
+        if(Input.GetMouseButton(0))
+        {
+            shootingStrength += 20f;
```

Comment:
Possibly place the shootingStrength logic in a FixedUpdate() instead of the Update() because Update() is based on framerate and better systems would have their arrow charge faster

Another option would be to add 3 different tiers of power that activate based on time passed.

Reply:
another option is to multiply the 20 with Time.deltaTime which the amount of time that has passed since the last frame. This solves the same issue where faster framerates won't lead to faster charging